driver-core/src/main/com/mongodb/client/gridfs/model/GridFSDownloadByNameOptions.java

```diff
@@ -22,7 +22,9 @@
  * <p>Controls the selection of the revision to download</p>
  *
  * @since 3.1
+ * @deprecated use {@link GridFSDownloadOptions} instead.
  */
+@Deprecated
```

Comment:
Even with the overloads, this class still only applies to downloading by name, so I'd also be ok leaving it alone and not deprecating it.  But I'll defer to you on it.